src/ciphersuite/mod.rs

```diff
@@ -56,6 +56,24 @@ pub struct HpkeCiphertext {
     ciphertext: Vec<u8>,
 }
 
+/// A label which can be used to expand a `Secret` using an HKDF.
+struct HkdfLabel {
+    length: u16,
+    label: String,
+    context: Vec<u8>,
+}
+
+impl HkdfLabel {
+    pub fn new(context: &[u8], label: &str, length: usize) -> Self {
+        let full_label = "mls10 ".to_owned() + label;
+        HkdfLabel {
+            length: length as u16,
```

Comment:
Good catch. This should not be the case if the ciphersuites are sane (i.e. if the `hash_length` isn't too large), but it should be checked and cought anyway. I've made the exporter throw an error if the key length is too large (this one actually comes from the outside) and would otherwise wait for the next `Secret` refactoring (issue #228), as we really ought to check the length systematically anyway.